source/libraries/ContractInterfacesGenerator.ts

```diff
@@ -169,6 +169,9 @@ ${contractMethods.join("\n\n")}
             case 'uint256[]': {
                 return 'Array<BN>';
             }
+            case 'bytes32[]': {
+                return 'Array<string>';
```

Comment:
Was this added to this PR opportunistically, or is it necessary as part of this PR?  I don't mind, just want to make sure I understand how it is or isn't associated with the rest of the changes.

Reply:
I'm actually uncertain how tests passed without this. Noted in my description that some changes here were to get tests passing